parts/22-query.sh

```diff
@@ -4905,3 +4905,67 @@ query_tpt-tool-memory() { ##? [--startyear=<YYYY>] [--endyear=<YYYY>] [--formula
 	EOF
 }
 
+query_tools-usage-per-month() { ##? [--startmonth=<YYYY>-<MM>] [--endmonth=<YYYY>-<MM>] [--toolnames=<tool1>,<tool2>,...]: toolnames is required. startmonth is 1 year ago and end month is current month by default
```

Comment:
```suggestion
query_tools-usage-per-month() { ##? [--startmonth=<YYYY>-<MM>] [--endmonth=<YYYY>-<MM>] [--tools=<tool1>,<tool2>,...]: toolnames is required. startmonth is 1 year ago and end month is current month by default
```

i think 'tools' would be more consistent across other commands. Also if it's mandatory then we should handle it differently. 

I think it might be nice if it was optional? I can imagine some admins for smaller galaxies just want *all* data?

Reply:
Yes, then I need to change this part.